### PR TITLE
Update en.json - SMTP Max. connections info

### PR DESCRIPTION
**Pull request by MaximilianKohler**
Addresses https://github.com/knadh/listmonk/issues/2069

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -426,7 +426,7 @@
     "settings.mailserver.idleTimeout": "Idle timeout",
     "settings.mailserver.idleTimeoutHelp": "Time to wait for new activity on a connection before closing it and removing it from the pool (s for second, m for minute).",
     "settings.mailserver.maxConns": "Max. connections",
-    "settings.mailserver.maxConnsHelp": "Maximum concurrent connections to the server.",
+    "settings.mailserver.maxConnsHelp": "Maximum concurrent connections to the server. Should = Message rate x Concurrency.",
```
Comment (knadh) [inline i18n/en.json:429]: This isn't correct. Max conns can be anything and don't need to be message_rate x concurrency.

Comment (MaximilianKohler) [reply i18n/en.json:429]: I based it on your comment here https://github.com/knadh/listmonk/issues/2069#issuecomment-2394927415

>your SMTP pool's max conns is set to 10, which means at max, you can only send 10 messages concurrently
>
>You should reduce concurrency to something like 5-10 (given you've sent max messages to 2) and increase SMTP max conns to match.

Would it be more accurate to say `SMTP Max. connections should = the Performance -> Concurrency number`?

Comment (MaximilianKohler) [reply i18n/en.json:429]: Your last comment in that issue also makes it sound like this current help text is incorrect:

>If concurrency = 10 and message_rate = 10, then up to 10x10=100 messages may be pushed out every second

Per your comment, it sounds like "Message rate" is the actual limit, and "Concurrency" just enables it.

So I think the help text for each of the performance settings needs some tweaking/clarification.

Comment (knadh) [reply i18n/en.json:429]: I think we should change the setting's label to `Max message rate` instead of `Message rate`.

Comment (MaximilianKohler) [reply i18n/en.json:429]: How would you be getting 50k/minute with 

> Concurrency: 250
> Message rate: 50

50/s x 60 seconds = 3k/minute.

Comment (knadh) [reply i18n/en.json:429]: In the instance of the settings I shared (which keep getting tweaked based on campaigns) 250 workers with each worker doing a max of 50/sec = 12.5k/sec max. For larger campaigns, we spawn and add multiple SMTP servers and raise the limits accordingly.

hm, sending 20/sec with 10 in settings, I'll have to investigate that.

Comment (MaximilianKohler) [reply i18n/en.json:429]: >250 workers with each worker doing a max of 50/sec = 12.5k/sec max

Ok, I guess there's a communication issue because this indicates that the `If concurrency = 10 and message_rate = 10, then up to 10x10=100 messages may be pushed out every second` help text is correct, even though [it sounded like you contradicted that recently](https://github.com/knadh/listmonk/issues/2069#issuecomment-2455468817).

Yet [my own experiments](https://github.com/knadh/listmonk/issues/2069#issuecomment-2452911380) indicate that help text is wrong.

* You're sending at 50,000 per minute
* 12500 per second is 750,000 per minute

None of this is making sense.

Comment (knadh) [reply i18n/en.json:429]: Lost track. After reviewing this, the help text is correct. It begins by saying `Max..`. So it's the maximum upper limit permitted, but it does not mean that the program can actually push out so many e-mails to an SMTP server. There are many bottlenecks (CPU/network resources, database bottlenecks, network, and of course, the capacity of the SMTP server itself).

We've set an upper limit of 250 workers with 50/worker/sec max. With that, including all the bottlenecks (primarily the capacity of the SMTP servers to handle messages), we're achieving a net throughput of 50k. If the SMTP servers aren't performant, it can drop to any number. It doesn't mean that the system will be achieve a 750k/min throughput.

Basically, like the help text indicates, it is the `Maximum` upper limit, not a guarantee of any throughput.
